TestTicketFlightsBooking/UnitTests.cs

```diff
@@ -0,0 +1,63 @@
+using AutoFixture;
+using FluentAssertions;
+using TicketFlightsBooking.Models;
+
+namespace TestTicketFlightsBooking
+{
+    public class UnitTests
```

Comment:
one of the most important benefits of unit testing is that modifications to modules that break the expected behaviour are caught earlier in the development cycle. for that, you need _ideally_ to test all modules even if they seem to be identical or replicated. for now, you don't have to change anything in this task, however, in the final project, I highly encourage to cover most scenarios (based on the priority of each scenario).